ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/JsonRpcHttpService.java

```diff
@@ -67,12 +67,15 @@
 import com.google.common.collect.Iterables;
 import io.netty.handler.codec.http.HttpResponseStatus;
 import io.opentelemetry.api.GlobalOpenTelemetry;
+import io.opentelemetry.api.baggage.propagation.W3CBaggagePropagator;
```

Comment:
W3C Baggage. 🤣

Reply:
You correctly understood the underlying mood of that W3C spec. For the idly interested, here is way more on the topic: https://www.w3.org/TR/baggage/